.travis.yml

```diff
@@ -38,19 +38,9 @@ env:
     - JHI_APP=ngx-psql-es-noi18n JHI_PROFILE=prod JHI_PROTRACTOR=1
     - JHI_APP=ngx-gradle-fr JHI_PROFILE=prod JHI_PROTRACTOR=1
     - JHI_APP=ngx-mariadb-oauth2-sass-infinispan JHI_PROTRACTOR=1
-    - JHI_APP=ngx-h2mem-ws-nol2
```

Comment:
May be we can later add more combinations here since we have free slots now

Reply:
yes totally !